recipes/simple-build-dgcode/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.0.12" %}
+
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: simple-build-dgcode
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/simple-build-dgcode/simple-build-dgcode-{{ version }}.tar.gz
+  sha256: fdde85d7b07dda6b11e6b28bcdd2510c45afdd2d976a096640cf86f63adc0a75
+
+build:
+  skip: True  # [win or py<39]
+  script: python -m pip install --no-deps -vv .
+  number: 0
+  entry_points:
+    - dgcode_newsimproject = simplebuild_dgcode._cli:newsimproject
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```

Comment:
```suggestion
```

I don't think we need to cross-compile anything here 🤔

Reply:
Right, I will remove them.